internal/builder/packages.go

```diff
@@ -210,32 +212,41 @@ func BuildPackage(ctx context.Context, options BuildOptions) (string, error) {
 	logger.Debug("Encode dashboards")
 	err = encodeDashboards(builtPackageDir)
 	if err != nil {
-		return "", fmt.Errorf("encoding dashboards failed: %w", err)
+		return "", nil, fmt.Errorf("encoding dashboards failed: %w", err)
 	}
 
 	logger.Debug("Resolve external fields")
 	err = resolveExternalFields(options.PackageRootPath, builtPackageDir)
 	if err != nil {
-		return "", fmt.Errorf("resolving external fields failed: %w", err)
+		return "", nil, fmt.Errorf("resolving external fields failed: %w", err)
 	}
 
 	err = addDynamicMappings(options.PackageRootPath, builtPackageDir)
 	if err != nil {
-		return "", fmt.Errorf("adding dynamic mappings: %w", err)
+		return "", nil, fmt.Errorf("adding dynamic mappings: %w", err)
 	}
 
 	err = resolveTransformDefinitions(builtPackageDir)
 	if err != nil {
-		return "", fmt.Errorf("resolving transform manifests failed: %w", err)
+		return "", nil, fmt.Errorf("resolving transform manifests failed: %w", err)
+	}
+
+	updatedReadmesTargets, err := readmeUpdater(options.RepositoryRoot, options.PackageRootPath, options.BuildDir)
```

Comment:
Could we directly use `docs.UpdateReadmes` instead of having to pass this function? At the end it makes sense to make the update of readmes part of the build.
```suggestion
	updatedReadmesTargets, err := docs.UpdateReadmes(options.RepositoryRoot, options.PackageRootPath, options.BuildDir)
```

Reply:
at the installer this step is not done, that is why i passed the function, and at the installer its a noop one to not run the update readme. 

we were discussing offline with @mrodm why the installer wasn't updating the readmes. some ideas where to decouple the build function and have one for installing a package and an other for building the package itself.

Reply:
updated on the last commit to pass a bool option.